hollow/core.py

```diff
@@ -621,7 +622,15 @@ def toarray(v):
   print "Excluding surrounded points"
   timer.start()
   hole_size = int(1.5 * 1.4 / grid_spacing)
-  grid.exclude_surrounded(hole_size)
+
+
+  print 'is_hollow has been set to: %s' % is_hollow
```

Comment:
A user has no idea what "is_hollow" means. Say in words what happens. 

Maybe refer to the `-i` option of hollow. But think about the messages from the user's perspective.